tempesta_fw/cache.c

```diff
@@ -487,7 +488,8 @@ node_db(void)
 static int
 tfw_cache_sched_cpu(TfwHttpReq *req)
 {
-	return c_nodes[req->node].cpu;
+	struct ca_node_t *node = &ca_nodes[req->node];
+	return node->cpu[node->cpu_idx++ % node->nr_cpus];
```

Comment:
I was thinking along the same lines, that strict atomicity is not necessary here.
Do you think it should be a strict atomic increment?